### PR TITLE
Add MAML doc

### LINKED ISSUE #2014: Docs Page "Algorithms and Methods -> Meta-RL -> MAML"

### PR DESCRIPTION
**Pull request by maliesa96**
closes #2014

### CODE REVIEW

File: docs/user/algo_maml.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+|                   |    :scale: 10%                                                                                                                                                             |
+|                   |    :class: no-scaled-link                                                                                                                                                  |
+|                   |                                                                                                                                                                            |
+|                   |    PyTorch                                                                                                                                                                 |
++-------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| **API Reference** | `garage.torch.algos.BC <../_autoapi/garage/torch/algos/index.html#garage.torch.algos.maml>`_                                                                               |
```
Comment (ryanjulian) [inline docs/user/algo_maml.md:13]: Update the paths in this and the next cell?

Comment (yeukfu) [inline docs/user/algo_maml.md:13]:
```suggestion
| **API Reference** | `garage.torch.algos.MAML <../_autoapi/garage/torch/algos/maml/index.html#garage.torch.algos.maml.MAML>`_                                                                               |
```

File: docs/user/algo_maml.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
++-------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| **Code**          | `garage/torch/algos/bc.py <https://github.com/rlworkgroup/garage/blob/master/src/garage/torch/algos/maml.py>`_                                                             |
```
Comment (yeukfu) [inline docs/user/algo_maml.md:15]:
```suggestion
| **Code**          | `garage/torch/algos/maml.py <https://github.com/rlworkgroup/garage/blob/master/src/garage/torch/algos/maml.py>`_                                                             |
```

**Review by yeukfu — APPROVED** [review]

File: docs/user/algo_maml.md (hunk 1 above)
Comment (maliesa96) [reply docs/user/algo_maml.md:13]: Good catch thank you!

**Review by ryanjulian — APPROVED** [review]

**Pull request merged**